$NetBSD: patch-aa,v 1.1.1.1 2009/06/01 21:41:39 seb Exp $

Prevent CPAN.pm to run an interactive setup process if it is
unconfigured: don't call Module::Install::auto_install()

--- Makefile.PL.orig	2009-05-10 14:45:08.000000000 +0000
+++ Makefile.PL
@@ -18,7 +18,7 @@ build_requires 'Authen::CAS::Client';
 build_requires 'URI::Escape';
 build_requires 'mod_perl2';
 
-auto_install;
+#auto_install;
 
 no_index directory => 'examples';
 #install_script('scripts/delete_session_data.pl');
